src/App.scss

```diff
@@ -1,11 +1,4 @@
 // Importing Global Variables
-@import "~@redhat-cloud-services/frontend-components-utilities/styles/_all";
-
-.widgetLayout {
```

Comment:
I don't think you want to delete this. Only the import.

Reply:
Fixed, sorry.